### ISSUE TITLE
Module reports only contains undefined

### ISSUE THREAD

**Issue by denniss17** [ember-cli/ember-modules-codemod#54]
I just ran the codemode over an application using ember 2.16.0-beta.2. In the end, this was the result:
```
All done. 
Results: 
0 errors
257 unmodified
0 skipped
244 ok
Time elapsed: 5.278seconds 

Done! Some files could not be upgraded automatically. See MODULE_REPORT.md.
```

However, when I open the `MODULE_REPORT.md` file, there are 51 of these exact entries:
```
### Unknown Global

**Global**: `Ember.undefined`

**Location**: `undefined` at line undefined

` ` `js
undefined
` ` `
```
(spaces added between "`")

Among others, I still see these kind of constructions:
```
const {
  String
} = Ember;
```
and
```
const {
  Logger
} = Ember;
```

Any clue why the report contains only undefined entries?

**Comment by backspace** [comment]
This happened to me too just now, though I only had two copies of the `undefined`. I searched the codebase for `Ember.` and theorise that it’s `Ember.testing`, which I have twice in the repository. I assume this can’t yet be handled in the module imports fashion…

It’s an open source repository, so it should be possible to reproduce with backspace/prison-rideshare-ui@f2e64cb. I didn’t run `ember-modules-codemod` directly but as part of `ember-cli-update` but I think that doesn’t matter…?

**Comment by kitsunde** [comment]
Same issue on:

```
import Mixin from '@ember/object/mixin';
import Ember from 'ember';

export default Mixin.create(Ember.TargetActionSupport, {});
```

**Comment by localpcguy** [comment]
Looks like I was getting this via `const {warn} = Ember.Logger;`.

**Comment by nickschot** [comment]
I was getting these on 
- Ember.Logger (Ember.Logger.debug)
- Ember.Handlebars (Ember.Handlebars.makeBoundHelper) (<- this one might be a residual on my part from an older Ember version?)

**Comment by Turbo87** [comment]
@rwjblue @Serabe could this be related to the latest updates regarding the new rfc-data format?

**Comment by rwjblue** [comment]
Unsure if it is directly related, but the issue here is that any globals in Ember not found in the mapping seem to be reported as Ember.undefined.

**Comment by efx** [comment]
Seeing this as well using ember-cli@2.16.2:
```
All done. 
Results: 
0 errors
372 unmodified
0 skipped
642 ok
Time elapsed: 12.789seconds 
```

**Comment by rwjblue** [comment]
The fix was just merged and released as 0.2.10.
